sdcm/utils/loader_utils.py

```diff
@@ -117,6 +117,47 @@ def _run_all_stress_cmds(self, stress_queue, params):
 
         return stress_queue
 
+    @staticmethod
+    def parse_cs_user_profiles_param(cs_user_profiles):
+        user_profiles_def = list(zip(*[profile.split(",") for profile in cs_user_profiles]))
+        user_profiles = user_profiles_def[0]
+        durations = None if len(user_profiles_def) == 1 else user_profiles_def[1]
```

Comment:
what it's durations ? in plural ? what would be the behavior is someone would pass in `user-profile.yaml,45m,66m` ?

I think we should leave to only have one option there for now, not more than that, and adapt the rest of the code to no use plurals here, it's confusing

Reply:
> what it's durations ? in plural ? what would be the behavior is someone would pass in `user-profile.yaml,45m,66m` ?
`66m` will be ignored

> I think we should leave to only have one option there for now, not more than that, and adapt the rest of the code to no use plurals here, it's confusing

Can you elaborate, please, what do you mean

Reply:
see the usage of this:
```
        user_profiles, durations = self.parse_cs_user_profiles_param(prepare_cs_user_profiles)
        stress_before_upgrade, _ = self.run_cs_user_profiles(cs_profiles=user_profiles, duration=durations)
```
one function return `durations` and the other name the argument `duration`, it's confused me
to the level I have to run the code to understand which one is right.

```python
>>> cs_user_profiles = ['ppp.yaml,60m', 'aaa.yaml,5m', 'aaa.yaml,10m,20m']
>>> user_profiles_def = list(zip(*[profile.split(",") for profile in cs_user_profiles]))
>>> user_profiles_def
[('ppp.yaml', 'aaa.yaml', 'aaa.yaml'), ('60m', '5m', '10m')]
>>>
```

so run_cs_user_profiles argument `duration`, should be rename to `durations` casue it's expect to be a list.
and `parse_cs_user_profiles_param` could use some docstring to explain the output, and what is the format it's support parsing